src/Stormpath/Resource/FacebookProvider.php

```diff
@@ -70,14 +69,4 @@ public function setClientSecret($clientSecret)
         $this->setProperty(self::CLIENT_SECRET, $clientSecret);
     }
 
-    public function getRedirectUri()
```

Comment:
Please add a note to the Changelog that we are removing this property because it has never been necessary, whenever you add the changes to that file.